lua/lvim/plugins.lua

```diff
@@ -4,25 +4,25 @@ local commit = {
   cmp_luasnip = "d6f837f4e8fe48eeae288e638691b91b97d1737f",
   cmp_nvim_lsp = "ebdfc204afb87f15ce3d3d3f5df0b8181443b5ba",
   cmp_path = "c5230cb439df9547294678d0f1c1465ad7989e5f",
-  comment = "90df2f87c0b17193d073d1f72cea2e528e5b162d",
-  dapinstall = "568d946a99edb6780912cb39ca68c368516cd853",
+  comment = "7365bfe9fc6dc004cc97b8977aa8129999d81bf5",
+  dapinstall = "24923c3819a450a772bb8f675926d530e829665f",
   dashboard_nvim = "d82ddae95fd4dc4c3b7bbe87f09b1840fbf20ecb",
   fixcursorhold = "0e4e22d21975da60b0fd2d302285b3b603f9f71e",
-  friendly_snippets = "a8efb538f4be58b4f4586bc964f9f10496c05e31",
-  gitsigns = "f43cee333fdfeb27aae86bab73a38c5885d5c7f4",
+  friendly_snippets = "4f6bd8eb5fbe0e45d57e732783ead2c3a01f549c",
+  gitsigns = "4a2d30f5fb77750c7a42be9bb58a9cc2c6c7f31d",
   lua_dev = "a0ee77789d9948adce64d98700cc90cecaef88d5",
-  lualine = "70691ae350fdbe1f15758e3b8e2973742a7967a9",
-  luasnip = "0222ee63c9e4b80e6000d064f8efd8edcc6d0c48",
+  lualine = "9208bae98fd5d1ab6145868a8c48bfee53c1a499",
+  luasnip = "59576a5cf28556a393eedfe38467e998288fc905",
   nlsp_settings = "3a3942b5d1da30e3ca0dc431aada3191c5952054",
-  null_ls = "0adceda4fb3fdd25ff015e68ebe176268b12bb2c",
+  null_ls = "59067dae4bf2367eb06326e419c23353722ecbec",
   nvim_autopairs = "97e454ce9b1371373105716d196c1017394bc947",
-  nvim_cmp = "7cb14b0c0f926e3cd5ac4d2e19634f82534c9d3c",
+  nvim_cmp = "d93104244c3834fbd8f3dd01da9729920e0b5fe7",
   nvim_dap = "c9a58267524f560112ecb6faa36ab2b5bc2f78a3",
-  nvim_lsp_installer = "cc34adb58de9e5a42af730c97ec04fc500bb5e63",
-  nvim_lspconfig = "ea29110765cb42e842dc8372c793a6173d89b0c4",
+  nvim_lsp_installer = "88e44bbbe2e03523e08f2bd3c9a954675bc438f4",
+  nvim_lspconfig = "c51096481dc13193991571b7132740d762902355",
   nvim_notify = "15f52efacd169ea26b0f4070451d3ea53f98cd5a",
-  nvim_tree = "0a2f6b0b6ba558a88c77a6b262af647760e6eca8",
-  nvim_treesitter = "9e90866bcea3e392126c4eb90fde56b1c0b56b69",
+  nvim_tree = "2dfed89af7724f9e71d2fdbe3cde791a93e9b9e0",
```

Comment:
👀

Reply:
have you tried the keybindings tho? 
https://github.com/LunarVim/lunarvim/blob/25e8b2234915f7862d1c6fcd9477b0c1b1ea1bf4/lua/lvim/core/nvimtree.lua#L129-L136
vs
https://github.com/kyazdani42/nvim-tree.lua/blob/2dfed89af7724f9e71d2fdbe3cde791a93e9b9e0/README.md#settings

Reply:
It does seem to work, since it's probably handling the deprecation more gracefully this time 🙏 

We would have to fix this at some point eventually tho 😅